mmedit/datasets/pipelines/augmentation.py

```diff
@@ -964,11 +966,13 @@ def __call__(self, results):
         lq_path_root = results['lq_path']
         gt_path_root = results['gt_path']
         lq_path = [
-            osp.join(lq_path_root, clip_name, f'{v:08d}.png')
+            osp.join(lq_path_root, clip_name,
+                     f'{self.filename_tmpl.format(v)}.png')
```

Comment:
Question: should `.png` be part of the template?

Reply:
We usually use `.png`, but I think that could work if someone wants to use `jpg` as inputs. I will modify that.